.github/labeler.yml

```
@@ -0,0 +1,19 @@
+frontend:
+- changed-files:
+  - any-glob-to-any-file: 'frontend/**/*'
+  - any-glob-to-any-file: 'frontend/**/*.js'
+  - any-glob-to-any-file: 'frontend/**/*.ts'
+  - any-glob-to-any-file: 'frontend/**/*.tsx'
+  - any-glob-to-any-file: 'frontend/**/*.jsx'
+  - any-glob-to-any-file: 'frontend/**/*.css'
+  - any-glob-to-any-file: 'frontend/**/*.html'
+
+backend:
+  - changed-files:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 0 but found 2

(indentation)